src/mgmt/srv.c

```diff
@@ -202,6 +202,38 @@ process_startms_request(Drpc__Call *drpc_req, Mgmt__DaosResponse *daos_resp)
 	}
 }
 
+static void
+process_getattachinfo_request(Drpc__Call *drpc_req,
+			      Mgmt__GetAttachInfoResp *resp)
+{
+	Mgmt__GetAttachInfoReq	*pb_req = NULL;
+	int			rc;
+
+	mgmt__get_attach_info_resp__init(resp);
+
+	/* Unpack the daos request from the drpc call body */
+	pb_req = mgmt__get_attach_info_req__unpack(
+		NULL, drpc_req->body.len, drpc_req->body.data);
+
+	if (pb_req == NULL) {
+		resp->status = MGMT__DAOS_REQUEST_STATUS__ERR_UNKNOWN;
+		D_ERROR("Failed to extract request\n");
+
+		return;
+	}
+
+	/* response status is populated with SUCCESS on init */
```

Comment:
I know this was copied from previous implementation but this comment applies to line 212 above, just a minor note

Reply:
Thanks for the approval and the reminder. I've created https://jira.hpdd.intel.com/browse/DAOS-2554 to track this issue.